RetailCoder.VBE/API/ParserState.cs

```diff
@@ -14,7 +14,56 @@
 namespace Rubberduck.API
 {
     [ComVisible(true)]
-    public interface IParserState
+    public enum ParserState
```

Comment:
I'd explicitly number any enums that are ComVisible, and assign ParserState.None zero.